test/functional/ex_cmds/write_spec.lua

```diff
@@ -100,11 +109,19 @@ describe(':write', function()
     funcs.setfperm(fname, 'r--------')
     eq('Vim(write):E505: "Xtest-functional-ex_cmds-write" is read-only (add ! to override)',
        exc_exec('write'))
-    os.remove(fname)
-    os.remove(fname_bak)
-    write_file(fname_bak, 'TTYX')
-    lfs.link(fname_bak .. ('/xxxxx'):rep(20), fname, true)
-    eq('Vim(write):E166: Can\'t open linked file for writing',
-       exc_exec('write!'))
+    if helpers.iswin() then
+      eq(0, os.execute('del /q/f ' .. fname))
+      eq(0, os.execute('rd /q/s ' .. fname_bak))
+    else
+      eq(true, os.remove(fname))
+      eq(true, os.remove(fname_bak))
+    end
+    eq(true, write_file(fname_bak, 'TTYX'))
+    -- FIXME: exc_exec('write!') outputs 0 in Windows
+    if not helpers.iswin() then
```

Comment:
just `return` early, to avoid the extra nesting and VCS churn.

Reply:
`if helpers.iswin() then return end`?